_lessons/16-setting-up-the-layout.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Setting up the Layout
+lesson: 16
+layout: default
+---
+
+Welcome to your first project! From this point on, there will be no more "Try it Yourself" boxes in the tutorial. Instead, you should follow along with everything on your own computer. You can also download the files for each project step on Github. 
+
+The lessons for this project will be a bit longer than in the previous sections, so be sure to pace yourself and take all the time you need to follow along. 
+
+### Set up the Project File
+
+The first thing we'll do, before we start writing any project code, is to create an HTML file with places to put our HTML and CSS. To recap, here is the standard structure of an HTML file:
+
+```
+<!DOCTYPE html>
+<html>
+  <head>
+    <style type="text/css">
+      (Your CSS goes here!)
+    </style>
+  </head>
```

Comment:
Maybe also add a `<title>` tag? Realistically most "real" webpages are going to want one.